packages/botonic-plugin-intent-classification/src/model/model-selector.ts

```diff
@@ -9,11 +9,13 @@ export class IntentModelSelector extends ModelSelector<
   IntentClassifier,
   IntentClassifierConfig
 > {
-  static async build(
-    locales: Locale[],
-    modelsBaseUrl: string
-  ): Promise<IntentModelSelector> {
-    const selector = new IntentModelSelector(locales, modelsBaseUrl)
+  static async build(locales: Locale[]): Promise<IntentModelSelector> {
+    const baseUrl: string =
+      // @ts-ignore
+      (typeof MODELS_BASE_URL !== 'undefined' && MODELS_BASE_URL) ||
+      process.env.MODELS_BASE_URL
```

Comment:
why not just use `const baseUrl: string = process.env.MODELS_BASE_URL` (simpler, cleaner code) and in our local configuration use the EnvironmentPlugin:
```
new webpack.EnvironmentPlugin({
  MODELS_BASE_URL: '...',
});

```

Reply:
definitely is not working in this way. We have tried.
```
new webpack.EnvironmentPlugin({
  MODELS_BASE_URL: '...',
});
```
and also
```
new webpack.DefinePlugin({
  'process.env.MODELS_BASE_URL': '...',
});
```
we should think carelessly what should be the best way to handle this kind of variables with newer webpack versions